packages/app/src/utils/tooltips.ts

```diff
@@ -0,0 +1,46 @@
+export const tooltips = {
+  yourStake: `Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy 
+  eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. 
+  At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, 
+  no sea takimata sanctus est Lorem ipsum dolor sit amet.`,
+
+  totalStatke: `Lorem ipsum dolor sit amet, consetetur sadipscing elitr, 
+  sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam 
+  erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea 
+  rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.`,
+
+  offsetCO2: `Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam 
+  nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam 
+  voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita 
+  kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.`,
+
+  withdraw: `Delayed withdrawal is always free. You will receive a ticket and at the
+  end of the epoch you can withdraw your SOL. Immediate withdrawal is
+  usually also free, but if the unstake amount is too large, a small fee
+  can be charged.`,
+
+  marinadeStakePool: `Delayed withdrawal is always free. You will receive a ticket and at the
+  end of the epoch you can withdraw your SOL. Immediate withdrawal is
+  usually also free, but if the unstake amount is too large, a small fee
+  can be charged.`,
+
+  marinadeLiquidityPool: `Delayed withdrawal is always free. You will receive a ticket and at the
+  end of the epoch you can withdraw your SOL. Immediate withdrawal is
+  usually also free, but if the unstake amount is too large, a small fee
+  can be charged.`,
+
+  solblazeStakePool: `Delayed withdrawal is always free. You will receive a ticket and at the
+  end of the epoch you can withdraw your SOL. Immediate withdrawal is
+  usually also free, but if the unstake amount is too large, a small fee
+  can be charged.`,
+
+  inflight: `Delayed withdrawal is always free. You will receive a ticket and at the
+  end of the epoch you can withdraw your SOL. Immediate withdrawal is
+  usually also free, but if the unstake amount is too large, a small fee
+  can be charged.`,
+
+  extractableYield: `Delayed withdrawal is always free. You will receive a ticket and at the
```

Comment:
```suggestion
  extractableYield: `The proportion of funds that have been accrued by Sunrise and are ready to be spent on retiring carbon credits. Once this balance reaches a limit, it will be extracted by the [Yield Controller](https://docs.sunrisestake.com/#sunrise-stake-treasury-spending) and used to purchase and burn carbon tokens.
```